packages/server/__snapshots__/7_record_spec.coffee.js

```diff
@@ -763,7 +763,22 @@ exports['e2e record api interaction errors uploading assets warns but proceeds 1
 exports['e2e record misconfiguration errors and exits when no browser found 1'] = `
 Can't run because you've entered an invalid browser name.
 
-Browser: 'browserDoesNotExist' was not found on your system.
+Browser: 'browserDoesNotExist' was not found on your system or is not supported by Cypress.
+
+Cypress supports the following browsers:
+- Canary
+- Chrome
+- Chromium
+- Edge
```

Comment:
the user is passing a machine-name to `--browser`, but now this error message is listing off display names, which might be confusing?

like, what if they install Edge and then try `--browser Edge` based off this error message - i believe it will fail since it will not match any `name` listed in `browsers.ts`